Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -1190,6 +1190,10 @@ localizedStrings["Observer Callback"] = "Observer Callback";
 localizedStrings["Observer Handlers:"] = "Observer Handlers:";
 localizedStrings["Observers:"] = "Observers:";
 localizedStrings["Off"] = "Off";
+/* The "on" state of switch controls. */
+localizedStrings["On @ Switch State"] = "On";
+/* The "off" state of switch controls. */
+localizedStrings["Off @ Switch State"] = "Off";
```

Comment:
this does not match what's actually in the code

FYI this file is intended to be autogenerated from running
```bash
./Tools/Scripts/extract-localizable-js-strings --utf8 ./Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js ./Source/WebInspectorUI/UserInterface
```